core/src/main/scala/org/apache/spark/util/RpcUtils.scala

```diff
@@ -54,6 +56,12 @@ private[spark] object RpcUtils {
     RpcTimeout(conf, Seq(RPC_LOOKUP_TIMEOUT.key, NETWORK_TIMEOUT.key), "120s")
   }
 
+  /**
+   * Infinite timeout is used internally, so there's no actual timeout property controls it.
+   * And timeout property should never be accessed since infinite means we never timeout.
```

Comment:
I'm not sure I follow this sentence correctly, can you try and reword it?

Reply:
Updated.

`timeout property` actually is the third field of `RpcTimeout`. You could check its usage inside `RpcTimeout`.